controllers/applicationconnector_controller_test.go

```diff
@@ -83,6 +88,10 @@ func testInstance(t time.Duration, ac v1alpha1.ApplicationConnector) {
 	}
 	Expect(k8sClient.Create(ctx, &gardenerCM)).Should(BeNil())
 
+	By(fmt.Sprintf("create compass-rt-agent configuration: %s/compass-agent-configuration", ac.Namespace))
```

Comment:
I don't think this is worth shortening, especially in logs...
```suggestion
	By(fmt.Sprintf("create compass-runtime-agent configuration: %s/compass-agent-configuration", ac.Namespace))
```